src/package.xml

```diff
@@ -2004,6 +2097,7 @@
         <members>Allocation__c</members>
         <members>Allocations_Settings__c</members>
         <members>Batch_Data_Entry_Settings__c</members>
+        <members>Batch_Template__c</members>
```

Comment:
Since this is a new object, you also need to add object translations. See https://salesforce.quip.com/bMdeAP7JiU0s